Flow.Launcher.Plugin/Result.cs

```diff
@@ -29,6 +31,8 @@ public class Result
         /// </summary>
         public string ActionKeywordAssigned { get; set; }
 
+        public string SuggestionText { get; set; } = string.Empty;
```

Comment:
No need to add a default here since you are already checking for null and string empty below.
```suggestion
        public string SuggestionText { get; set; }
```